### PR TITLE
Dd/login tests

### PR DESCRIPTION
**Pull request by DavidDufresne**

### CODE REVIEW

**Comment by DavidDufresne** [comment]
@joelhawksley What do you think of this?

File: features/support/roles/hive_user.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+module Roles
+  class HiveUser < Dill::Role
+    widget :github_signin_link, ['a', text: 'Sign in with Github']
+    widget :signout_link, ['a', text: 'Sign out']
```
Comment (joelhawksley) [inline features/support/roles/hive_user.rb]: Can we use class selectors vs. matching on strings?

Comment (DavidDufresne) [inline features/support/roles/hive_user.rb]: We could.

**Comment by joelhawksley** [comment]
:+1:

**Pull request merged**